config/variables/CanadaPubSecALZ-main.yml

```diff
@@ -0,0 +1,78 @@
+variables:
+  var-hubnetwork-region: canadacentral
+  var-hubnetwork-managementGroupId: Connectivity
+  var-hubnetwork-azfw-configurationFileName: hub-azfw/hub-network.parameters.json
+  var-hubnetwork-nva-configurationFileName: hub-nva/hub-network.parameters.json
+  var-hubnetwork-subscriptionId: 4fd845de-f6c8-4e6d-9a87-c21c4ebf7edd
+  var-hubnetwork-azfwPolicy-configurationFileName: hub-azfw-policy/azure-firewall-policy.parameters.json
+  var-logging-managementGroupId: Management
+  var-logging-subscriptionId: 91e1aaa2-a0a0-4770-8d90-02daa39bf57a
```

Comment:
Since we'll be applying the `logging` portion of the LZ, does this need to update to our current logging subscription ID?

Reply:
Yes, I updated it in the new commit